rollup.yaml

```diff
@@ -360,7 +374,7 @@ components:
         - input_index
         - block_number
         - block_timestamp
-        - payload
+        - prev_randao
```

Comment:
We don't have an epoch_index anymore, right? Are we supposed to keep this field there? @miltonjonat @fabiooshiro

Reply:
If it is always zero, it is not really deprecated. If somebody accesses it, it is better for it to crash than for it to appear 0 when the app expected otherwise.

Reply:
We are not uisng it @miltonjonat @diegonehab .